magenta/music/midi_io_test.py

```diff
@@ -344,6 +344,30 @@ def testIsDrumDetection(self):
           channel_counts[index] += 1
     self.assertEqual(channel_counts, [2, 2])
 
+  def testInstrumentInfo_NoteSequenceToPrettyMidi(self):
+    source_sequence = music_pb2.NoteSequence()
+    source_sequence.notes.add(pitch=60, start_time=0.0,
+                              end_time=0.5, velocity=80, instrument=0)
+    source_sequence.notes.add(pitch=60, start_time=0.5,
+                              end_time=1.0, velocity=80, instrument=1)
+    instrument_info1 = source_sequence.instrument_infos.add()
+    instrument_info1.name = 'inst_0'
+    instrument_info1.instrument = 0
+    instrument_info2 = source_sequence.instrument_infos.add()
+    instrument_info2.name = 'inst_1'
+    instrument_info2.instrument = 1
+    translated_midi = midi_io.sequence_proto_to_pretty_midi(source_sequence)
+    translated_sequence = midi_io.midi_to_note_sequence(translated_midi)
+
+    if translated_sequence.instrument_infos:
```

Comment:
I think you don't want this if statement because you want the assertions to fail if instrument_infos is not present.

Reply:
Done!